backend/dotnet/Meowmentum.Server.Dotnet.Api/Meowmentum.Server.Dotnet.Api/Controllers/TaskContoller.cs

```diff
@@ -0,0 +1,64 @@
+﻿using AutoMapper;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Meowmentum.Server.Dotnet.Business.Abstractions;
+using Meowmentum.Server.Dotnet.Shared.Requests.Task;
+using Meowmentum.Server.Dotnet.Api.Helpers;
+using Task = Meowmentum.Server.Dotnet.Core.Entities.Task;
+
+namespace Meowmentum.Server.Dotnet.Api.Controllers;
+
+[Route("api/[controller]")]
+[ApiController]
+[Authorize]
+[ValidateModel]
+public class TasksController(ITaskService taskService, IMapper mapper) : BaseController()
+{
+    [HttpPost("create")]
```

Comment:
lets work with standart rest conventions
https://medium.com/@nadinCodeHat/rest-api-naming-conventions-and-best-practices-1c4e781eb6a5
so remove `create` from route